packer/conf/buildkite-agent/scripts/buildkite-agent-lifecycled-handler

```diff
@@ -11,7 +11,7 @@ for i in $(seq 1 "${BUILDKITE_AGENTS_PER_INSTANCE}"); do
   service "buildkite-agent-${i}" stop &
 
   # Need to ensure it's the buildkite-agent user, so it doesn't match this lifecycld handler script
-  while pgrep -u buildkite-agent buildkite-agent > /dev/null; do
+  while pgrep -u buildkite-agent buildkite-agent-${i} > /dev/null; do
```

Comment:
Nice, thanks! Good catch. Can you add double quotes around that just for bash listing and it's good for merge party?